images/disks-images-provider/entrypoint.sh

```diff
@@ -27,7 +27,10 @@ mkdir -p /images/datavolume1 /images/datavolume2 /images/datavolume3
 echo "converting cirros image from qcow2 to raw, and copying it to local-storage directory, and creating a loopback device from it"
 # /local-storage will be mapped to the host dir, which will also be used by the local storage provider
 qemu-img convert -f qcow2 -O raw /images/cirros/disk.img /local-storage/cirros.img.raw
+
+# Put LOOP_DEVICE in /etc/bashrc in order to detack this loop device when the pod stopped.
 LOOP_DEVICE=$(losetup --find --show /local-storage/cirros.img.raw)
+echo LOOP_DEVICE=${LOOP_DEVICE} >> /etc/bashrc
```

Comment:
I don't understand what's the goal of this ... You're writing the name of the loop device to the bashrc in the container . What are you planning to do with that ?

Reply:
It is used in the hook to delete the specific device.
See https://github.com/kubevirt/kubevirt/pull/3555#discussion_r441417636

Reply:
Ah! My bad! Thanks.